cluster/manifests/skipper/service.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     prometheus.io/scrape: "true"
 spec:
   type: ClusterIP
+  clusterIP: 10.3.99.99
```

Comment:
I don't think you can change `clusterIP` of a service.

Reply:
good point